content/post/11-Eliminating-Distractions/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Eliminating Distractions"
+date: 2010-11-13T15:56:33-07:00
```

Comment:
Now that I think about it, we may as well set the seconds to zero, so can you adjust this date as follows?

```suggestion
date: 2010-11-13T00:00:00-07:00
```